docs/source/developer-guides/prs-and-reviews.rst

```diff
@@ -0,0 +1,75 @@
+Consolidation:
+==============
+
+Overarching Goals of good Review
+--------------------------------
+
+The goal of a review is to ensure that the proposed change is useful and maintainable over the long run.
+
+PR Description
+---------------
+
+- Have a meaningful title: it's easier when creating the changelog or when searching through old PR
+- Motivate why we should merge the PR: adding/changing code risks introducing a new bug. IMO one person asking for a nice to have feature doesn't qualify.
+- Explain the PR does in the description: it makes it easier/faster to review
+- Make the PR as small as possible: it's much easier to review five 200 lines PR than one 1000 lines PR
+- If a PR is known to create conflict with other active PRs, try to coordinate, and link to each other
+  - if appropriate explain desired review/merge order 
```

Comment:
```suggestion
  - if appropriate, explain the desired review/merge order 
```

Reply:
- In the case of conflicting PR, would this be too much to ask non-team members to manage / understand, particularly given the relatively great likelihood of some number of PR for View (for instance) having merge conflicts?
- Instead, would this make sense:  first, ask the potential PR author to check the PR queue to see if their desired change is already reflected in an existing PR; if not, then create a new PR with the (focused / "surgical" / single) desired change; if so, participate in the review of the existing PR, thereby helping move PR through the queues, and formulate ideas for future changes